src/main/java/io/confluent/examples/streams/ApplicationResetExample.java

```diff
@@ -161,7 +161,7 @@ public static KafkaStreams run(final String[] args, final Properties streamsConf
     final KafkaStreams streams = new KafkaStreams(builder.build(), streamsConfiguration);
 
     // Delete the application's local state on reset
-    if (args.length > 0 && args[0].equals("--reset")) {
+    if (args.length > 1 && args[1].equals("--reset")) {
```

Comment:
I think it would be better to make the example more flexible. IMHO, this method could just accept a `boolean doReset` instead of `String[] args` and we should add code in `main()` method, to check for 3 cases (zero, one, two arguments passed) and calls `run()` accordingly.

Reply:
indeed. Would be much better. I will update the PR.

Reply:
It might also be better to parse options with something like commons-cli. Currently the first argument is expected to be bootstrap-servers, and the second one the reset option. With named arguments the order would matter.

Reply:
We could. IMHO, it's ok for this example to keep it simple.